estimagic/dashboard/monitoring_app.py

```diff
@@ -47,8 +58,13 @@ def monitoring_app(
     database = load_database(path=session_data["database_path"])
     start_point = _calculate_start_point(database, rollover, jump)
     session_data["last_retrieved"] = start_point
-    start_params, group_to_params = _get_group_to_params_from_database(database)
-    criterion_history, params_history = _create_cds_for_monitoring_app(start_params)
+    start_params = load_start_params(path_or_database=database)
+    start_params["id"] = _create_id_column(start_params)
+    group_to_param_ids = _map_groups_to_param_values(start_params, "id")
```

Comment:
`_map_group_to_param_values` is not very good, because it can be any column and rarely will be the value column. 

`_map_group_to_other_column` would be better.

Reply:
Done in a675a68.